src/components/ebay-video/README.md

```diff
@@ -52,8 +55,9 @@ If multiple sources are supplied, it will go in the order they are supplied and
 
 ## ebay-video events
 
-| Event        | Data                        | Description           |
-| ------------ | --------------------------- | --------------------- |
-| `load-error` | `{ originalEvent }`         | error occurred        |
-| `play`       | `{ originalEvent, player }` | play start            |
-| `report`     | `{ }`                       | report button clicked |
+| Event          | Data                               | Description                                                                          |
+| -------------- | ---------------------------------- | ------------------------------------------------------------------------------------ |
+| `load-error`   | `{ originalEvent }`                | error occurred                                                                       |
+| `play`         | `{ originalEvent, player }`        | play start                                                                           |
+| `volumechange` | `{ originalEvent, volume, muted }` | volume is changed. Will return the values of the volume decimal and muted true/false |
```

Comment:
Name should be separated with a dash?

Reply:
This is one that I'm not sure about. 
The html5 event is without a dash and non camel case. (Which is what this is and how i kept it)
Which is why i kept it as is. I can rename it to be with a dash though which is more consistent with our codebase

Reply:
Yeah, let's keep it consistent with our side.